spec/transform.spec.js

```diff
@@ -13,6 +13,9 @@ var dummySchema = {
         return 'geopoint';
       }
       return;
+    },
+    getRelationFields: function() {
```

Comment:
Nope, we need it on the dummy schema otherwise the function is missing when running the tests